packages/web-pkg/src/components/FilesList/ResourceTiles.vue

```diff
@@ -363,10 +380,31 @@ export default defineComponent({
       return resource.processing === true
     }
 
+    const disabledResources: ComputedRef<Array<Resource['id']>> = computed(() => {
```

Comment:
Can we name this `disabledResourceIds`? Otherwise it suggests full resources here. Then you could also remove the type annotation `ComputedRef<Array<Resource['id']>>` and let TS infer the type.

Reply:
Sure this was just copy paste